src/lib.rs

```diff
@@ -95,10 +96,23 @@ impl Builder {
     ///
     ///`tag` - Event category to send for each record.
     pub fn new(tag: &'static str) -> Self {
+        const DEFAULT_MAX_MSG_RECORD: usize = 10;
         Self {
             tag,
             writer: default,
             fmt: NestedFmt,
+            max_msg_record: DEFAULT_MAX_MSG_RECORD,
+        }
+    }
+
+    #[inline(always)]
+    ///Provides max message record to fetch up.
+    pub fn with_max_msg_record(self, max_msg_record: usize) -> Self {
```

Comment:
`max_msg_record` should be `core::num::NonZeroUsize` in function argument instead of `usize` as `0` would make no sense as argument here

Reply:
Fix: b88f107